src/WebJobs.Script.WebHost/App_Data/secrets/host.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy",
-  "functionKey": "zlnu496ve212kk1p84ncrtdvmtpembduqp25ajjc"
+    "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy,z3dlq50s00cb3q3k11nil7xyt29ebst2n8rtn0t3",
```

Comment:
I actually didn't refer back to #270, and simply didn't have the extra scenario in mind. If we're talking about giving access to separate parties (basically multiple sas keys), then I see how the dictionary can make sense. I think we should dicuss if we could make that work without to much extra work. e.g. we'd need to come up with a default key name when we first create the key, but that's easy enough (call it "default").